protocol/shannon/fullnode_cache.go

```diff
@@ -34,138 +33,136 @@ const (
 	// TODO_MAINNET_MIGRATION: Ensure applications are invalidated during unstaking. Revisit these values after mainnet migration to ensure no race conditions.
 	defaultAppCacheTTL = 5 * time.Minute
 
-	// Cleanup interval is twice the TTL.
-	// standard practice to balancing cleanup with minimal processing overhead
-	defaultAppCacheCleanupInterval = defaultAppCacheTTL * 2
-
 	// As of #275, on Beta TestNet.
 	// - Blocks are 30 seconds
 	// - A session is 50 blocks
 	// - A grace period is 1 block (i.e. 30 seconds)
 	// TODO_MAINNET_MIGRATION: Revisit these values after mainnet migration to ensure no race conditions.
 	defaultSessionCacheTTL = 30 * time.Second
-	
-	// Cleanup interval is twice the TTL.
-	// Standard practice for balancing cleanup with minimal processing overhead
-	defaultSessionCacheCleanupInterval = defaultSessionCacheTTL * 2
 
-	// Cache key prefixes to avoid collisions
+	// Early refresh configuration for apps
+	// Apps will start refreshing when they're between 4-4.5 minutes old
```

Comment:
1. #PUC what "refreshing" means and why we chose this number
2. Please add TODO_TECHDEBT(@olshansk) to revisit this.